class RepoError(Exception):
    pass


class ValidationError(Exception):
    pass


